06.bowling_object/game.rb

```diff
@@ -5,27 +5,24 @@
 
 class Game
   def initialize(input_text)
-    marks = input_text.split(',')
+    shots = input_text.split(',').map { |mark| Shot.new(mark) }
     count = 0
+    frame_index = 0
     @frames = []
-    while count < marks.size && @frames.size < 9
-      shot = Shot.new(marks[count])
-      if shot.strike?
-        @frames << Frame.new(shot)
-        count += 1
-      else
-        @frames << Frame.new(shot, Shot.new(marks[count + 1]))
-        count += 2
-      end
+    while count < shots.size && @frames.size < 9
```

Comment:
```suggestion
    while @frames.size < 9
```

これだとうまく動かないケースってあります？

Reply:
不要な条件でしたので削除しました。